tests/test_scenarios.py

```diff
@@ -29,16 +30,15 @@ def test_scenario(filename, monkeypatch):
     command, output = load_scenario(path)
 
     with runner.isolated_filesystem():
-        subprocess.check_call(["git", "init"])
-        Path("a").mkdir()
-        Path("b/c").mkdir(parents=True)
-        Path("a/pyproject.toml").touch()
-        Path("b/c/build.gradle").touch()
-        # TODO the commit here is necessary; project finding only works based
-        # on files that are present in the original repo, and this doesn't work
-        # with NullRepo either. Oops.
-        subprocess.check_call(["git", "add", "-N", "."])
-        subprocess.check_call(["git", "commit", "-a", "-m", "foo"])
+        repo_data = path.parent / "repo"
+        if repo_data.exists():
```

Comment:
is the intent to have more than one repo dir, and it uses the most specific?  I'd expect this to walk up, or default to the SCENARIO_DIR / "repo" worst case.

Reply:
I figured there would be one "repo" dir per scenario subdirectory, but we can brainstorm how this should work.